src/pages/VenuePage/VenuePage.tsx

```diff
@@ -71,8 +70,8 @@ export const VenuePage: React.FC = () => {
 
   // const [isAccessDenied, setIsAccessDenied] = useState(false);
 
-  const { user, profile } = useUser();
-  const { userLocation } = useWorldUserLocation(user?.uid);
+  const { user, profile, userLocation } = useUser();
+  console.log(userLocation);
```

Comment:
Remove

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2292/commits/18636351f55c7dccea7213907110d8a0cec1d106